src/sass/_hca.scss

```diff
@@ -0,0 +1,216 @@
+// Variables
```

Comment:
Hmm, I didn't mean to change anything here, so I'm not sure why it's showing up in the diff.